flow/endpoint.yaml

```diff
@@ -70,6 +70,15 @@ components:
           enum: [mesh, one_to_one]
           default: mesh
         tx_names:
+          description: |-
+            The system will use information from the supplied tx_names to populate the following
+            flow.packet.header fields if they are present:
+              - ethernet.dst using device ipv4 addresses from either interfaces, route ranges etc.
+              - ethernet.src using ethernet.mac addresses from ethernet interfaces
+              - ipv4.src using device ipv4 addresses from either interfaces, route ranges etc.
```

Comment:
Do we need to document this now -  Those are convenience features that can be looked at later.

Reply:
Those implicit actions are already being done in other implementations (snappi[ixnetwork]) so they should be documented now so there is no confusion at a later date.

Reply:
And users of snappi[ixnewtork] are already expecting that behavior.